src/hdmf/common/resources.py

```diff
@@ -126,6 +152,7 @@ def __init__(self, **kwargs):
         super().__init__(name)
         self.keys = kwargs['keys'] or KeyTable()
         self.resources = kwargs['resources'] or ResourceTable()
+        self.entities = kwargs['entities'] or EntityTable()
         self.objects = kwargs['objects'] or ObjectTable()
         self.object_keys = kwargs['object_keys'] or ObjectKeyTable()
 
```

Comment:
Reviewing this code again, can you docval-ize the `add_key` method on line 159 -- change key to **kwargs and add docval to it, like the other methods. Then the `key` argument which maybe should be renamed `key_name` can have a docstring and type associated with it.